api/src/wsmanager.py

```diff
@@ -18,6 +19,10 @@ def is_valid_uuid(uuid_string):
     return val.hex == uuid_string.replace('-', '')
 
 
+def ignore_none(items: List[Tuple[str, Any]]) -> dict:
```

Comment:
lol I thought you said this one-liner was too ugly

Reply:
It is kinda gross.